tools/trace/inspect/Main.cpp

```diff
@@ -0,0 +1,108 @@
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include <iostream>
+
+#include "llvm/Support/CommandLine.h"
+#include "llvm/Support/Error.h"
+#include "llvm/Support/ToolOutputFile.h"
+
+#include "revng/PipelineC/Tracing/Trace.h"
+#include "revng/Support/CommandLine.h"
+#include "revng/Support/InitRevng.h"
+
+namespace cl = llvm::cl;
```

Comment:
I'd avoid introducing namespace aliases, just add `using namespace`, unless you have a reason not to.

Reply:
Dropped